tests/unit/bases/test_buildd.py

```diff
@@ -42,11 +42,9 @@
 
 @pytest.fixture()
 def mock_load(mocker):
-    mocker.patch(
+    yield mocker.patch(
```

Comment:
out of curiosity, why `yield` instead of `return`ing the patch object here?

Reply:
No good reason.  Most of the pytest fixtures yield instead of return, so I've been blindly following that practice.

Updated.